assets/sass/protocol/components/_button.scss

```diff
@@ -135,8 +135,7 @@ a.mzp-c-button {
 // Primary
 
 .mzp-c-button,
-a.mzp-c-button:link,
-a.mzp-c-button:visited {
+a.mzp-c-button {
```

Comment:
The increased specificity was added in https://github.com/mozilla/protocol/pull/1045 to address the problem where links were styled like buttons and then got focus.

This has been solved by removing the text colour change for :focus styles from the default :link styles.